greenkeeper.json

```diff
@@ -6,6 +6,7 @@
         "packages/extension-dapp/package.json",
         "packages/extension-ui/package.json",
         "packages/extension/package.json"
+        "packages/ui/package.json",
```

Comment:
Should this be `packages/extension-polymath-ui/package.json`? Shall we add the other package too? I'm not sure what is this honestly.